generator/typesystem_gui.xml

```diff
@@ -917,10 +914,6 @@
 
       <modify-function signature="operator*(QMatrix)const" access="private"/>
       <modify-function signature="operator*=(QMatrix)" access="private"/>
-      <modify-function signature="rotate(double)" access="private" rename="rotate_private"/>
-      <modify-function signature="scale(double,double)" access="private" rename="scale_private"/>
-      <modify-function signature="shear(double,double)" access="private" rename="shear_private"/>
-      <modify-function signature="translate(double,double)" access="private" rename="translate_private"/>
```

Comment:
What have these been for? Is this pre-Qt5 stuff?

Reply:
I don't know, at least since Qt5 these variants don't even exist (the arguments are all float now according to the documentation). And PythonQt should have no problems to discern between float and double variants of the same method.